src/Integration.UnitTests/Persistence/SolutionBindingTests.cs

```diff
@@ -169,6 +189,12 @@ public void SolutionBinding_ReadSolutionBinding_NoFile()
             this.outputPane.AssertOutputStrings(0);
         }
 
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", 
+            "S1854:Dead stores should be removed", 
+            Justification = "False positive, stream is used", 
+            Scope = "member", 
+            Target = "~M:SonarLint.VisualStudio.Integration.UnitTests.SolutionBindingTests.SolutionBinding_ReadSolutionBinding_IOError")]
```

Comment:
The rule isn't complaining that the stream object isn't used: it's saying that the assignment to the _stream_ variable isn't necessary since the _stream_ variable isn't used.
An object doesn't have to be assigned to a variable for the _using_ block to dispose it correctly.